test/integration/test_koji.py

```diff
@@ -37,18 +24,71 @@ def koji_command(*args, _input=None, _globals=None, **kwargs):
                           check=False)
 
 
-def parse_os_release():
-    info = {}
-    with open("/etc/os-release", encoding="utf-8") as f:
-        for line in f:
-            line = line.strip()
-            if not line:
-                continue
-            if line[0] == "#":
-                continue
-            k, v = line.split("=", 1)
-            info[k] = v.strip('"')
-    return info
+class SutInfo:
+    """Class representing information about the system under test"""
+
+    REPOS = {
+        "fedora": [
+            {"url": "http://download.fedoraproject.org/pub/fedora/linux/releases/$release/Everything/$arch/os"}
+        ],
+        "rhel": [
+            {"url": "http://download.devel.redhat.com/released/RHEL-8/$release/BaseOS/$arch/os/",
+            "package_sets": "blueprint; build; packages"},
+            {"url": "http://download.devel.redhat.com/released/RHEL-8/$release/AppStream/$arch/os/",
+            "package_sets": "blueprint; build; packages"},
+        ]
+    }
+
+    def __init__(self):
+        info = self.parse_os_release()
```

Comment:
This uses `self.` but `parse_os_release` is a `@staticmethod`. It feels non-idiomatic, but I'm not that experienced in Python to judge.

Reply:
So static method does not rely on the state of the object nor the class, but it is always an attribute of both. I could reference it using the class name, but it feels silly to me. I'm happy to do a small research to learn what is idiomatic.

Reply:
So it seems that calling a static method on the class object is preferred to make it easier to sub-class the class. A class inheriting from this one could override the method and calling it on `self` would result in calling the overridden method. It is not an issue in this case, but I'll rework it.